jasmine-ajax/jasmine-ajax.d.ts

```diff
@@ -73,6 +73,9 @@ declare class MockAjax {
 
 	stubRequest(url: RegExp, data?: string, method?: string): JasmineAjaxRequestStub;
 	stubRequest(url: string, data?: string, method?: string): JasmineAjaxRequestStub;
+    
+	stubRequest(url: RegExp, data?: RegExp, method?: string): JasmineAjaxRequestStub;
+	stubRequest(url: string, data?: RegExp, method?: string): JasmineAjaxRequestStub;
 
```

Comment:
When overload arguments only differ only by type, take advantage of union types. The 4 overloads can be condensed into:

    stubRequest(url: string | RegExp, data?: string | RegExp, method?: string): JasmineAjaxRequestStub;